<launch>
    <node pkg="fusion_slam" type="runMapping" name="fusion_slam" output="screen" /> 
</launch>